[13.0][FIX] helpdesk_mgmt: portal access error.

victoralmau opened the pull request:
When the user has no access rights the `search` call returns an AccessError.

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT38713

victoralmau committed:
[FIX] helpdesk_mgmt: portal access error.

When the user has no access rights the `search` call returns an AccessError.
TT38713

pedrobaeza commented:
One thing: why using `_prepare_portal_layout_values` instead of `_prepare_home_portal_values` that is used by standard modules instead?

victoralmau committed:
[IMP] helpdesk_mgmt. Replace to the correct function.

Replace _prepare_portal_layout_values to _prepare_home_portal_values function.
TT38713

victoralmau commented:
I have applied the change to the correct function (there seems to be no reason to use the other even though they are very similar).

pedrobaeza commented:
I think in v12 there's no such difference. Both works, but this one avoids to gather the ticket count each time a portal page is rendered. Thanks for the changes!

/ocabot merge patch
